spec/lib/percy/capybara/loaders/base_loader_spec.rb

```diff
@@ -14,6 +14,18 @@
       expect(resource.content).to include('Hello World!')
       expect(resource.sha).to eq(Digest::SHA256.hexdigest(resource.content))
     end
+    it 'includes an iframe resource' do
```

Comment:
To be clear, I'm talking about a _new_ test — this one is fine as is, just want to add a different one to make sure RackTests still run if there is an iframe in the HTML.